src/util/rangePolygon.js

```diff
@@ -15,7 +15,7 @@ export default function rangePolygon (chart){
             return lbornrhi !== 'NA' ? chart.y(+lbornrhi) : 0
         });
 
-    var dRow = chart.filtered_data[0];
+    var dRow = chart.raw_data[0];
 
```

Comment:
Just kidding, you're absolutely right.  Will update to `this.filtered_data` in a new branch.